delatore/sources/awx_api.py

```diff
@@ -42,12 +42,18 @@ class AWXApiSource(Source):
     _params: Optional[AWXParams] = None
 
     async def get_update(self):
-        template_name = await self.client.get(self.polling_interval)
-        if template_name is None:
+        template_name_and_depth = await self.client.get(self.polling_interval)
+        if template_name_and_depth is None:
             raise NoUpdates
-
+        split_template = template_name_and_depth.split(';')
+        depth = 1
+        if len(split_template) == 2:
+            template_name = split_template[0]
+            depth = split_template[1]
+        else:
+            template_name = template_name_and_depth
```

Comment:
Seems a bit hard to read, why not smth like `name, *depth = ...` than handle cases when depth is empty and when it is not